x-pack/filebeat/module/fortinet/clientendpoint/test/generated.log-expected.json

```diff
@@ -0,0 +1,5702 @@
+            "10.84.105.75"
+        ],
+        "destination.port": 98,
+        "event.action": "deny",
+        "event.code": "ms-wbt-server",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "August 30 15:48:33 aperia4409.www5.invalid proto=rdp service=ms-wbt-server status=deny src=10.78.151.178 dst=10.84.105.75 src_port=1846 dst_port=98 server_app=uames pid=499 app_name=msequi traff_direct=isnostru block_count=1559 logon_user=deFinibu@iadese6958.www5.local msg=unknown",
+        "event.outcome": "Failure",
+        "fileset.name": "clientendpoint",
+        "host.name": "aperia4409.www5.invalid",
+        "input.type": "log",
+        "log.offset": 4017,
+        "network.direction": "isnostru",
+        "network.protocol": "rdp",
+        "observer.product": "FortiClient",
+        "observer.type": "Anti-Virus",
+        "observer.vendor": "Fortinet",
+        "process.pid": 499,
+        "related.ip": [
+            "10.84.105.75",
+            "10.78.151.178"
+        ],
+        "related.user": [
+            "deFinibu"
+        ],
+        "rsa.counters.dclass_c1": 1559,
+        "rsa.counters.dclass_c1_str": "block_count",
+        "rsa.internal.messageid": "ms-wbt-server",
+        "rsa.investigations.ec_outcome": "Failure",
+        "rsa.investigations.ec_subject": "NetworkComm",
+        "rsa.investigations.ec_theme": "ALM",
+        "rsa.misc.action": [
+            "deny"
+        ],
+        "rsa.misc.result": "unknown",
+        "rsa.network.alias_host": [
+            "aperia4409.www5.invalid"
+        ],
+        "rsa.network.domain": "iadese6958.www5.local",
+        "rsa.network.network_service": "ms-wbt-server",
+        "rsa.time.event_time": "2019-08-30T17:48:33.000Z",
+        "server.domain": "iadese6958.www5.local",
+        "service.type": "fortinet",
+        "source.ip": [
+            "10.78.151.178"
+        ],
+        "source.port": 1846,
+        "tags": [
+            "fortinet.clientendpoint",
+            "forwarded"
+        ],
+        "user.name": "deFinibu"
+    },
+    {
+        "@timestamp": "2019-09-14T00:51:07.000Z",
+        "destination.ip": [
+            "10.76.229.163"
+        ],
+        "destination.port": 6387,
+        "event.action": "deny",
+        "event.code": "ms-wbt-server",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "September 13 22:51:07 tatn2376.www5.corp proto=ipv6-icmp service=ms-wbt-server status=deny src=10.47.241.218 dst=10.76.229.163 src_port=2890 dst_port=6387 server_app=CSed pid=2857 app_name=utlabore traff_direct=ecillu block_count=391 logon_user=mnisist@sedd3727.api.home msg=unknown",
+        "event.outcome": "Failure",
+        "fileset.name": "clientendpoint",
+        "host.name": "tatn2376.www5.corp",
+        "input.type": "log",
+        "log.offset": 4299,
+        "network.direction": "ecillu",
```

Comment:
is this just from munging the data or do we need to do some additional normalization? `ecillu` doesn't seem like a valid direction

Reply:
This is the log generator not knowing how to populate direction fields. I'll update it.